corehq/apps/couch_sql_migration/management/commands/migrate_domain_from_couch_to_sql.py

```diff
@@ -343,3 +321,14 @@ def _blow_away_migration(domain, dst_domain):
     sql_case_ids = CaseAccessorSQL.get_deleted_case_ids_in_domain(dst_domain)
     CaseAccessorSQL.hard_delete_cases(dst_domain, sql_case_ids)
     log.info("blew away migration for domain {}\n".format(domain))
+
+
+def _iter_migrated_cases(domain):
+    case_ids = chain(
+        CaseAccessorSQL.get_case_ids_in_domain(domain),
+        CaseAccessorSQL.get_deleted_case_ids_in_domain(domain)
```

Comment:
Not sure if it matters for the size of domains you're migrating, but it's generally better to use paging when loading all case ids in a domain. This will load them all into memory at once, which could take a long time and/or use a lot of memory on large domains.

Reply:
Both those methods return lists. I could use a generator expression or `partial` so that only one list gets loaded at a time. Do you think it's worth it? I don't think any of the domains have a lot of deleted cases, so I'm not sure it'll make much difference.

Reply:
(fwiw, I tried creating an `CaseAccessorSQL.iter_case_ids_in_domain()` method but the cursor gets closed before the downstream call finishes iterating ... which explains why the method doesn't exist already.)

Reply:
Yeah, I was thinking something more along the lines of [`paginate_query_across_partitioned_databases`](https://github.com/dimagi/commcare-hq/blob/3a11df4c568be44b315e477d45a1d3cd39ac109a/corehq/sql_db/util.py#L72) (for SQL) or something like [`_iter_changes`](https://github.com/dimagi/commcare-hq/blob/48ae68bebdac3b323da1851504d8b9d7e5aa5d4c/corehq/apps/couch_sql_migration/couchsqlmigration.py#L833-L839) (for Couch).

Reply:
Nice. Thank you.